test/test_onnx.py

```diff
@@ -103,6 +103,21 @@ def forward(self_module, images):
         input_test = [torch.rand(3, 800, 1280), torch.rand(3, 800, 800)]
         self.run_model(TransformModule(), [input, input_test])
 
+    @unittest.skip("Disable test until Resize opset 11 is implemented in ONNX Runtime")
```

Comment:
Would there be a way of partially testing that the behavior is correct? Even if it's not end-to-end, but to at least be able to verify that some other parts of the code (which do not depend on Resize) work on ONNX?

Reply:
I tested locally by replacing bilinear interpolate by nearest interpolate, and was able to get a ONNX model that matches numbers with the PyTorch model.

Reply:
Could you test `FasterRCNN` part of `roi_heads` (which doesn't use `interpolate`)? Then we could add afterwards tests for the `maskrcnn_inference` part, once ONNX interpolate supports bilinear.

At least we would have the box detection path which would be tested.

Reply:
Makes sense. I will add a test very soon